frontend/src/components/posting/edit.js

```diff
@@ -60,6 +60,8 @@ export default class extends Form {
   }
 
   loadSuccess = (data) => {
+    this.state.originalPost = data.post
```

Comment:
Question: does this need to be a state var? Can't we put this into class property?

Reply:
Sure thing will update.